.github/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 <h3>New features since last release</h3>
 
+* Added Reset error to noise channels. 
```

Comment:
It might be better here to provide the name of the new operation, so that users can find it easily 🙂 
```suggestion
* Added a new noise channel, `qml.ResetError`.
```

Reply:
In addition @nahumsa -- feel free to add an example of it's usage here (check out some of the other changelog entries to see how the examples are formatted).